### DISCUSSION TITLE
Validation data is now available!!

### DISCUSSION THREAD

**Discussion by sarthakpati** [FETS-AI/Challenge#29] [General]
Hello All,

The training and validation data is now available! Please [register](https://miccai2021.fets.ai/) to receive the download link and further instructions.

Cheers,
Organizing Team

**Comment by dskhanirfan** [comment]
Dear Organizers,
Thanks
Validation Data is available, However, is there any code available to evaluate the performance metrics on validation dataset?

**Reply by sarthakpati** [reply]
>  is there any code available to evaluate the performance metrics on validation dataset?

This will be done via our evaluation application, which will be made available through our Image Processing Portal later this week.

**Reply by zhanghaoyue** [reply]
currently the IPP platform asks us to upload nii.gz format of validation result. Are we expected to write our own test code to perform the prediction or a standardized notebook will be provided?
Is there a ranking board at this moment?

Thank you!

**Reply by sarthakpati** [reply]
> Are we expected to write our own test code to perform the prediction 

Tagging @brandon-edwards @msheller @psfoley @alexey-gruzdev 

> Is there a ranking board at this moment?

Not at the moment, we are currently working towards this, and will update participants soon!

**Reply by brandon-edwards** [reply]
> currently the IPP platform asks us to upload nii.gz format of validation result. Are we expected to write our own test code to perform the prediction or a standardized notebook will be provided?

We will provide updated code with this feature. An update will come within the next few days, and I will also post to this thread to notify you directly.

**Reply by brandon-edwards** [reply]
@zhanghaoyue sorry for the delay, validation code was just merged. The notebook has additional cells (at the bottom of the notebook) now that take you through the process to create the folder of best model inference outputs.

**Reply by Linardos** [reply]
The provided validation code seems to do inference on the validation data, but is there a script to also obtain and store Dice and Hausdorff metrics from it?

**Comment by eceisik** [comment]
Dear Organizers,
do we have a daily quota for validation job submission? I remembered that I saw three job submission limit but I couldn't find the explanation again.
Best,
Ece

**Reply by sarthakpati** [reply]
Hey @eceisik,

There is no limit on the number of **validation** jobs being submitted.

**Reply by Linardos** [reply]
Hmm, to me it says that I have reached my quota (10/10 Jobs). And also it seems like the validation jobs today are running particularly slow:
![image](https://user-images.githubusercontent.com/26694607/126657607-a89d39f2-0865-46d5-8614-0e80c3a8365e.png)

Any ideas?

**Reply by sarthakpati** [reply]
Tagging @ujjwalbaid0408

**Reply by dskhanirfan** [reply]
My Jobs are 6/10, I have the same problem

**Reply by ujjwalbaid0408** [reply]
You can delete the completed jobs. We are receiving a high volume of jobs for validation and it would take some time for your job to finish. Please let us know if this is still pending for the next 48 hours.

**Reply by Linardos** [reply]
Hmm, it's been 24 hours+ without progress still. I wanted to point out that it's not "pending" it's been stuck on the "running" stage as described in the platform:
![image](https://user-images.githubusercontent.com/26694607/126778074-944da516-dd8d-4d8f-bc23-0a6a3c24f708.png)

**Reply by ujjwalbaid0408** [reply]
Hi Akis,

We are looking into this and will get back to you asap. If you receive any notification from IPP please let us know.

**Reply by ujjwalbaid0408** [reply]
Hi @Linardos,

Is the issue resolved now?

**Comment by zhanghaoyue** [comment]
I uploaded validation data from the output folder in separated files, names are like this : FeTS21_Vlidation_001_seg.nii.gz..I do check the segmented results visually and they are there. However when I submitted the result to IPP I got this:
core-FeTS21eval_validationPhase.execlist.1 log:
No Matching Ground Truth Files Found! Exiting.

**Reply by dskhanirfan** [reply]
Are you using the 3rd link on ipp portal?

**Reply by zhanghaoyue** [reply]
yes

**Comment by dskhanirfan** [comment]
<img width="433" alt="Screenshot 2021-07-13 at 11 18 04" src="https://user-images.githubusercontent.com/46925837/125417927-5c82a239-2684-4795-ab0f-1618f0f29ec5.png">
I can see three options on ipp portal. Can you kindly provide the link where I can upload my validation data that everyone is using? Is it the 3rd link in the pic?

**Reply by mzenk** [reply]
Yes. 
As I'm not sure about why there is a "training data" job here, I'll tag @sarthakpati , though.

**Reply by dskhanirfan** [reply]
That was confusing me actually

**Reply by sarthakpati** [reply]
I believe the "training data" job is for sanity-checking. The "FeTS'21 Validation Data: Segmentation Task" job is for evaluating the score of the validation data.

**Reply by Linardos** [reply]
Followed instructions faithfully (used the validation snippet on the jupyter notebook to infer the ground truths then uploaded them on the ipp portal using option 3 as indicated here). The job just ended this is what I received in the logs:
![image](https://user-images.githubusercontent.com/26694607/125455151-2bbefd39-20b5-407f-9439-4e784b2d725a.png)

**Reply by sarthakpati** [reply]
Hi @Linardos, can you please drop a line to our email with an example of the segmentation file you used? Tagging @ujjwalbaid0408 for follow-up.

**Reply by zhanghaoyue** [reply]
I got the exact same issue

**Reply by sarthakpati** [reply]
@zhanghaoyue Can you confirm that you don't have "_seg" present in the filenames during submission?

**Reply by zhanghaoyue** [reply]
the file name like this "FeTS21_Validation_001_seg.nii.gz"

**Reply by sarthakpati** [reply]
Please remove the `"_seg"` in the filename and resubmit.

**Reply by shouldersLL** [reply]
The readme file of task 2 requires to use "_seg" for the name of predictions. Should we keep or remove it in our submitted container? @sarthakpati 

Many thanks.

**Reply by sarthakpati** [reply]
For Task-2, please keep the "_seg" nomenclature.

**Comment by Linardos** [comment]
Yes! Just saw the results. It's working now
